src/tables.css

```diff
@@ -31,7 +31,7 @@
  *
  */
 .table,
-.prose table {
+.prose table:not(.unprose) {
```

Comment:
I think you can create unprose containers, as well, by changing these selectors like so:

`prose table:not(.unprose, .unprose table)`

I think we should still recommend applying it *directly* to the elements that you're unprosing, because otherwise you could get yourself in a little bit of a confusing tangle about whether you're in a prose section or not. But do you think it's still worthwhile to add this capability?